app/controllers/course/achievement/condition/achievements_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Achievement::Condition::AchievementsController <
+  Course::Condition::AchievementsController
+  include Course::AchievementConditionalConcern
```

Comment:
I think I get you. You need this because you need the both the conditional and the condition.